### DISCUSSION TITLE
searxng.example.com instead of example.com/searxng

### DISCUSSION THREAD

**Discussion by gianmarcogg03** [searxng/searxng#1895] [Q&A]
How can I set a subdomain to show SearXNG rather than a global subdirectory on Apache? The documentation isn't very clear about it.

**Comment by FinlayDaG33k** [comment]
You can use a virtualhost for this.
https://httpd.apache.org/docs/2.4/vhosts/examples.html

**Comment by sduensin** [comment]
This is asked all over the net if you search for a solution.  All I've managed so far is redirect loops.  I'm trying to modify the socket example from the documentation.   Help?
